finat/sympy2gem.py

```diff
@@ -45,3 +45,8 @@ def sympy2gem_float(node, self):
 @sympy2gem.register(sympy.Symbol)
 def sympy2gem_symbol(node, self):
     return self.bindings[node]
+
+
+@sympy2gem.register(sympy.Rational)
+def sympy2gem_rational(node, self):
+    return gem.Literal(node.numerator()) / gem.Literal(node.denominator())
```

Comment:
If they're both literals constant-fold them: `return gem.Literal(node.numerator() / node.denominator())`

Reply:
But this seems oddly specific, it won't always be the case. So probably do
```
return gem.Division(self(node.numerator()), self(node.denominator()))
```

the `Division` node in gem already constant-folds literals.